routers/api/actorlist/Actorlist.js

```diff
@@ -1,28 +1,28 @@
-const models=require('../../../models');
-
-async function actorList(req,res,next)
-{
-    try{
+const models = require('../../../models');
+const Logger = require('../../../services/logger')
+const logger = new Logger('Actorlist')
+async function actorList(req, res, next) {
+    try {
         const actors = await models.MoviePersons.findAll({
-            where:{roleId:"1"},
-            group:["name","age"],
-            attributes:["name","age"]
-            
+            where: { roleId: "1" },
+            group: ["name", "age"],
+            attributes: ["name", "age"]
+        })
+        logger.info("sucessful get actorlist")
+        res.status(200).json({
```

Comment:
Write one common function for success and failure response. Call that method from every route by sending statusCode and result
And format will always be same.
Like
For success:{ sucess: true, data: result}
For failure:{success: false, error: actual error }

Reply:
done